web_src/js/features/admin-users.js

```diff
@@ -0,0 +1,32 @@
+export function initAdminUserListSearchForm() {
+  const searchForm = window.config.PageData.adminUserListSearchForm;
```

Comment:
use an identified div and get the data from `data-` attributes.

Reply:
Explained in previous comment. We need a clear method to pass complex data. HTML `data` doesn't work.